bigquery/tests/system.py

```diff
@@ -402,8 +402,7 @@ def test_load_table_from_local_avro_file_then_dump_table(self):
 
```

Comment:
Looks like there are no system tests that specifically target get_table? Maybe worth writing one to confirm that we really get all the fields.

Reply:
:+1: We should have at least one system test for each client method. Then again, any such system test will likely have to load a table first.

@lukesneeringer Do you know if  there's a script somewhere to setup test resources that we should update, if we wanted to have system tests for pure GET methods?

Reply:
I believe the system tests should be self-contained. Ideally each run will create a new uniquely named dataset and tables within it, so multiple runs can occur in parallel even within the same project.

Maybe you're uncomfortable that the get_table test depends on create_table, and the create_table test depends on get_table? Actually, maybe those can be the same test. Anyway, the circularity doesn't really bother me.

Reply:
In that case, maybe one of these load tests just needs a new name (and maybe some additional asserts) to explicitly call out that it is testing both load and get?

Reply:
I added a system test using a public dataset

Reply:
@alixhami Clever!